tests/installation/addon_products_sle.pm

```diff
@@ -78,8 +79,10 @@ sub handle_all_packages_medium {
     for my $i (@addons) {
         push @addons_license_tags, "addon-license-$i" if grep(/^$i$/, @addons_with_license);
         send_key 'home';
-        send_key_until_needlematch "addon-products-all_packages-$i-highlighted", 'down', 30;
-        send_key 'spc';
+        unless ($i =~ $sle_prod) {
```

Comment:
We should not put such a condition, you handle 15-sp2 case above already, so we can leave this code unchanged.

Reply:
b6b3e0280